_internals/configuration.md

```diff
@@ -118,329 +118,329 @@ When `ANSI_DEFAULTS` is enabled, it enables the following ISO settings:
+### `babelfishpg_tsql.escape_hatch_session_settings`
 
 Escape hatch for session settings.
 
 
-### babelfishpg_tsql.escape_hatch_storage_on_partition
+### `babelfishpg_tsql.escape_hatch_storage_on_partition`
 
-Escape hatch for `storage_on_partition` option in 
+Escape hatch for `storage_on_partition` option in
 `CREATE/ALTER TABLE` and `CREATE INDEX`.
 
 
-### babelfishpg_tsql.escape_hatch_storage_options
+### `babelfishpg_tsql.escape_hatch_storage_options`
 
 Escape hatch for storage options option in `CREATE/ALTER TABLE/INDEX`.
 
 
-### babelfishpg_tsql.escape_hatch_table_hints
+### `babelfishpg_tsql.escape_hatch_table_hints`
 
 Escape hatch for table hints.
 
 
-### babelfishpg_tsql.escape_hatch_unique_constraint
+### `babelfishpg_tsql.escape_hatch_unique_constraint`
 
 Escape hatch for unique constraints.
 
 
-### babelfishpg_tsql.fmtonly
+### `babelfishpg_tsql.fmtonly`
 
 SQL-Server compatibility `FMTONLY` option.
 
 
-### babelfishpg_tsql.implicit_transactions
+### `babelfishpg_tsql.implicit_transactions`
 
 Enable implicit transactions.
 If no transaction is running, start an implicit transaction
 for qualified commands when `implicit_transactions` config
 option is on.
 
 
-### babelfishpg_tsql.language
+### `babelfishpg_tsql.language`
 
-T-SQL compatibility `LANGUAGE` option.
+T-SQL compatibility language option.
 
 
-### babelfishpg_tsql.migration_mode
+### `babelfishpg_tsql.migration_mode`
 
-Defines is multiple databases are supported. Valid options are: 
-`single-db`, `multi-db` and `NULL` (same as `single-db`). 
+Defines if multiple databases are supported. Valid options are:
+`single-db`, `multi-db` and `NULL` (same as `single-db`).
 
 
-### babelfishpg_tsql.nocount
+### `babelfishpg_tsql.nocount`
 
 T-SQL compatibility `NOCOUNT` option
 
 
-### babelfishpg_tsql.noexec
+### `babelfishpg_tsql.noexec`
 
 SQL-Server compatibility `NOEXEC` option
 
 
-### babelfishpg_tsql.numeric_roundabort
+### `babelfishpg_tsql.numeric_roundabort`
 
-Ends a query when an overflow or divide-by-zero error occurs 
+Ends a query when an overflow or division-by-zero error occurs
 during query execution.
-`ON` setting is not allowed for this option. 
+The setting `ON` is not allowed for this option.
 Please use `babelfishpg_tsql.escape_hatch_session_settings` to ignore.
 
 
-### babelfishpg_tsql.quoted_identifier
+### `babelfishpg_tsql.quoted_identifier`
 
 Interpret double-quoted strings as quoted identifiers.
 
 
-### babelfishpg_tsql.rowcount
+### `babelfishpg_tsql.rowcount`
 
-Causes the DB engine to stop processing the query after the 
+Causes the DB engine to stop processing the query after the
 specified number of rows are returned.
 
 
-### babelfishpg_tsql.server_collation_name
+### `babelfishpg_tsql.server_collation_name`
 
 Name of the default server collation. The default value is
 `sql_latin1_general_cp1_ci_as`.
 
 
-### babelfishpg_tsql.showplan_all
+### `babelfishpg_tsql.showplan_all`
 
 SQL-Server compatibility `SHOWPLAN_ALL` option.
 
```

Comment:
no dash between SQL and Server (applies to many cases). product names must be referenced correctly.

Reply:
You are right; changed.